ui/server/src/test/resources/ui.conf

```diff
@@ -90,8 +90,9 @@ processConfig {
   }
 
   nodeCategoryMapping: {
```

Comment:
Hmm maybe nodeGroup is better name? We already use category in another meaning..

Reply:
I don't want to refactor in this change... also you have categories of sports, categories of movies, IMO it is not a bad name, but always should exist with name of thing that you want to categorize.

Reply:
It's only suggestion ;) I've already added approve :)